src/Draco.Compiler/Internal/Syntax/Parser.cs

```diff
@@ -1008,6 +1018,13 @@ private StringExpressionSyntax ParseMultiLineString()
         return elements.ToSeparatedSyntaxList();
     }
 
+    private SyntaxToken? ParseVisibilityModifier()
+    {
+        var peek = this.Peek();
+        if (peek == TokenKind.KeywordInternal || peek == TokenKind.KeywordPublic) return this.Advance();
```

Comment:
```suggestion
        if (IsVisibilityModifier(peek)) return this.Advance();
```
Heck, after this the whole thing can become a oneliner